ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/readme.adoc

```diff
@@ -0,0 +1,253 @@
+= ocp4_workload_example_shared_cluster - Example Workload Role deployed to Shared Clusters
+
+== Preparation to Use This Role
+
+=== Get Access to Shared Cluster and Create Quotas
+
+This example role assumes you'll be using the Shared Cluster provided by GPTE.
+There should be a catalog item in CloudForms to give you access and get stated developing.
+
+== Description
+
+This role teaches the use of agnosticD ocp4_workload roles on pre-existing OpenShift clusters where admin access to the cluster is not available.
+The user must have "self-provisioner" roles and quotas already setup by another process, usually labs.opentlc.com or rhpds.redhat.com.
```

Comment:
That's true for developing. But AgnosticV will run these roles as "system:admin"